style.css

```diff
@@ -82,3 +87,144 @@ body {
   display: inline;
   margin-right: 13px;
 }
+
+.container {
+  margin: 0 auto;
+  width: 85%;
+}
+
+.projects {
+  margin: 100px auto;
+  padding: 5%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: white;
+}
+
+.tonic-project-img {
+  width: 100%;
+}
+
+.project-info {
+  text-align: left;
+  font-family: "Poppins", sans-serif;
+}
+
+.project-name {
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.project-subtitle {
+  display: flex;
+  flex-direction: row;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: 600;
+  color: #7a869a;
+}
+
+.project-subtitle span {
+  border-radius: 50%;
+  background-color: #7a869a;
+  width: 8px;
+  height: 8px;
+  margin: auto 15px;
+}
+
+.project-description p {
+  font-size: 15px;
+  font-weight: 400px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.project-description button {
+  border: none;
+  border-radius: 5px;
+  padding: 4px 12px;
+  margin-right: 7px;
+  color: #6070ff;
+  background-color: #ebebff;
+}
+
+.regular-btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  padding: 12px;
+  background-color: transparent;
+  border: 1px solid #6070ff;
+  color: #6070ff;
+  margin-top: 25px;
+}
```

Comment:
- [ ] Nice work with the buttons, they look exactly like the Figma design :smile_cat: However you did follow the button styling: Active, Hover, Pressed (see design)
![Screenshot from 2023-03-24 18-26-25](https://user-images.githubusercontent.com/91142432/227609751-4e89e8ca-4102-4ee3-ac74-98bf04bd46f4.png)
Kindly implement it to follow the Figma design